.github/workflows/quality.yml

```diff
@@ -0,0 +1,32 @@
+name: Quality
+
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  eslint:
+    # Executes `lint:fix` to clean up errors or
+    # warnings that are autofixable.
+    # Also posts warnings or errors that are not autofixable.
+    name: Eslint Fix
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Install deps
+        run: yarn
+
+      - name: Run ESLint Fix
+        run: yarn lint:fix
+
+      - uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "chore: run lint:fix"
```

Comment:
I wasn't expecting a GH job for this but I think it is fine and covers the goal pretty well. How do you see adding prettier to it? since my idea was to create a precommit/prepush hook to deal with these two.

Reply:
We can certainly do a pre-commit instead. You could add a format script that should essentially do the same type of job run as `lint:fix`.

But if you prefer a pre-commit, I can remove this. I figure I would at least make sure I presented this as an option. :D

What was suggested at this point in doing the pre-commit/push hook?

Reply:
Ok cool, that is fine. Lets keep this job and see how it work for us.

I think this might have an advantage over the precommit hook: it will always catch & fix linting errors, whether the user checks them or not.

Reply:
Sounds good!